tests/cpp_unit_tests/test_power_sensor.cpp

```diff
@@ -101,13 +113,29 @@ TEST_CASE("Test power sensor") {
             sym_sensor_param = sym_power_sensor.calc_param<true>();
             asym_sensor_param = sym_power_sensor.calc_param<false>();
 
-            CHECK(sym_sensor_param.variance == doctest::Approx(4.0 / 1e2));
+            CHECK(sym_sensor_param.p_variance == doctest::Approx(4.0 / 1e2 / 2));
+            CHECK(sym_sensor_param.q_variance == doctest::Approx(4.0 / 1e2 / 2));
             CHECK(real(sym_sensor_param.value) == doctest::Approx(3.0 * 1e-3));
             CHECK(imag(sym_sensor_param.value) == doctest::Approx(4.0 * 1e-3));
 
-            CHECK(asym_sensor_param.variance == doctest::Approx(4.0 / 1e2));
+            CHECK(asym_sensor_param.p_variance[0] == doctest::Approx(4.0 / 1e2 / 2));
+            CHECK(asym_sensor_param.q_variance[1] == doctest::Approx(4.0 / 1e2 / 2));
             CHECK(real(asym_sensor_param.value[0]) == doctest::Approx(3.0 * 1e-3));
             CHECK(imag(asym_sensor_param.value[1]) == doctest::Approx(4.0 * 1e-3));
+
+            // Check update with different p and q sigma variances
```

Comment:
here and below, i did not add any tests for infinite sigmas cause I deemed that trivial. I can add them if you like